### PR TITLE
added addons to heroku app.json

### PR DESCRIPTION
**Pull request by beniwohli**

### CODE REVIEW

File: server/config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ var conf = module.exports = {
     port: process.env.PORT || 3001
   },
   pg: {
+    url: process.env.DATABASE_URL || null,
```
Comment (watson) [inline server/config.js]: No reason to add ` || null` here. I guess you copied it from the `redis` property, but it shouldn't have been there either

**Comment by watson** [comment]
What's the reason you removed the `OPBEAT_*` environment variables?

**Comment by piquadrat** [comment]
@watson I removed them because they are set by the `opbeat` heroku addon

**Comment by watson** [comment]
@piquadrat tests failed. If you remove the trailing `;`, then it should pass 👍

**Comment by watson** [comment]
@beniwohli ping 😃

**Comment by watson** [comment]
@beniwohli thanks 😃 Do you mind fixing the failing linter errors? Should be fairly simple. If not, let me know and I'll do it

**Pull request closed without merging**